package.json

```diff
@@ -17,15 +18,22 @@
     "babel-jest": "^22.4.3",
     "babel-preset-env": "^1.6.1",
     "babel-preset-react": "^6.24.1",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
     "jest": "^22.4.3",
     "jest-css-modules-transform": "^1.0.5",
     "react-test-renderer": "^16.3.1"
   },
+  "engines": {
+    "node": ">=6.14.0",
+    "npm": ">=3.10.10"
+  },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "jest",
-    "eject": "react-scripts eject"
+    "test": "jest -u",
```

Comment:
Hmmm... I am not sure this is the purpose of the -u.  I think that we need a case where the test runs and it checks against the existing snapshot and if it doesn't match it fails. But I'm not sure what the purpose of the snapshots is other than that... when we commit, we should make sure the snapshots were also committed, and the match should be valid without the -u.